cmd/admin-handlers.go

```diff
@@ -799,17 +883,32 @@ func (a adminAPIHandlers) SetConfigKeysHandler(w http.ResponseWriter, r *http.Re
 		return
 	}
 
-	origConfig := string(configBytes)
-	configStr := origConfig
+	configStr := string(configBytes)
+	var config = newServerConfig()
```

Comment:
It would be helpful to add a comment to explain why we initialize a newServerConfig() instead of declaring a simple variable of type serverConfig{}